lib/Bindings/Python/OMModule.cpp

```diff
@@ -121,6 +143,20 @@ struct Object {
     return omEvaluatorValueToPythonValue(result);
   }
 
+  // Get a tuple of the field and location from the Object.
+  std::pair<PythonValue, MlirLocation> getFieldAndLoc(const std::string &name) {
+    // Wrap the requested field name in an attribute.
+    MlirContext context = mlirTypeGetContext(omEvaluatorObjectGetType(value));
+    MlirStringRef cName = mlirStringRefCreateFromCString(name.c_str());
+    MlirAttribute nameAttr = mlirStringAttrGet(context, cName);
+
+    // Get the field's ObjectValue via the CAPI.
+    OMEvaluatorValue result = omEvaluatorObjectGetField(value, nameAttr);
+
+    return {omEvaluatorValueToPythonValue(result),
```

Comment:
IMHO we should just return the field's location here. It seems like the API is more orthogonal this way--you can iterate over an object to get its field names and field values. You can access a field value by name. You can access a field location by name. Does that make sense? I think in any situation where you'd want to get a field's location, you'd have the field name already.